package.json

```diff
@@ -1,102 +1,37 @@
-    "build:main": "cd ./packages/main && vite build",
-    "build:preload": "cd ./packages/preload && vite build",
-    "build:renderer": "cd ./packages/renderer && vite build",
-    "compile": "cross-env MODE=production npm run build && electron-builder build --config electron-builder.cjs --dir",
-    "compile:installer": "cross-env MODE=production npm run build && electron-builder build --config electron-builder.cjs --publish never",
-    "test": "npm run test:main && npm run test:preload && npm run test:renderer && npm run test:e2e && npm run test:shared",
-    "test:e2e": "npm run build && vitest run",
-    "test:main": "vitest run -r packages/main --passWithNoTests",
-    "test:preload": "vitest run -r packages/preload --passWithNoTests",
-    "test:renderer": "vitest run -r packages/renderer --passWithNoTests",
-    "test:shared": "vitest run -r packages/shared --passWithNoTests",
-    "watch": "node scripts/watch.js",
-    "start": "npm run watch",
-    "lint": "eslint . --ext js,mjs,cjs,ts,mts,cts,vue",
-    "typecheck:main": "tsc --noEmit -p packages/main/tsconfig.json",
-    "typecheck:preload": "tsc --noEmit -p packages/preload/tsconfig.json",
-    "typecheck:renderer": "tsc --noEmit -p packages/renderer/tsconfig.json",
-    "typecheck": "npm run typecheck:main && npm run typecheck:preload && npm run typecheck:renderer",
-    "postinstall": "cross-env ELECTRON_RUN_AS_NODE=1 electron scripts/update-electron-vendors.js",
-    "format": "npx prettier --write \"**/*.{js,ts,tsx,json}\""
+  "dependencies": {
+    "fast-deep-equal": "^3.1.3"
   },
   "devDependencies": {
-    "@dcl/hooks": "0.0.2",
-    "@reduxjs/toolkit": "2.2.6",
-    "@sentry/vite-plugin": "3.2.2",
-    "@sentry/wizard": "4.2.0",
-    "@types/cmd-shim": "5.0.2",
-    "@types/http-server": "0.12.4",
-    "@types/is-running": "2.1.2",
-    "@types/node": "20.14.9",
-    "@types/react": "18.3.3",
-    "@types/react-dom": "18.3.0",
-    "@types/redux-logger": "3.0.13",
-    "@types/segment-analytics": "0.0.38",
-    "@types/semver": "7.5.8",
+    "@dcl/ts-proto": "^1.154.0",
+    "@testing-library/react": "^14.0.0",
     "@typescript-eslint/eslint-plugin": "7.15.0",
-    "@vitejs/plugin-react": "4.3.1",
-    "ansi-to-html": "0.7.2",
-    "classnames": "2.5.1",
-    "cross-env": "7.0.3",
-    "dcl-catalyst-client": "21.8.1",
-    "decentraland-ui2": "0.23.1",
-    "electron": "37.2.3",
-    "electron-builder": "26.0.12",
     "eslint": "8.57.0",
     "eslint-plugin-import": "2.31.0",
     "eslint-plugin-vue": "9.27.0",
-    "flat": "6.0.1",
     "happy-dom": "14.12.3",
-    "nano-staged": "0.8.0",
     "playwright": "1.45.0",
     "prettier": "3.5.3",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
-    "react-intl": "6.6.8",
-    "react-redux": "9.1.2",
-    "react-router-dom": "6.24.1",
-    "redux-logger": "3.0.6",
-    "simple-git-hooks": "2.11.1",
+    "redux-saga-test-plan": "^4.0.6",
```

Comment:
do you know if we can move the not common dev dependencies to each package?

e.g. `redux-saga-test-plan` is only used by the inspector package